docs/system/k8s-metrics.md

```diff
@@ -1089,5 +1091,35 @@ This metric is [recommended][MetricRecommended].
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## K8s Container metrics
+
+**Description:** K8s Container level metrics captured under the namespace `k8s.container`.
```

Comment:
is this statement necessary?

Reply:
I'm following what we have for other namespaces like at https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/k8s-metrics.md?plain=1#L73.

If there is a reason to remove those statements we can do it in a separate PR?